test/parallel/test-sqlite-database-sync.js

```diff
@@ -324,3 +324,54 @@ suite('DatabaseSync.prototype.exec()', () => {
     });
   });
 });
+
+suite('DatabaseSync.prototype.location()', () => {
+  test('throws if database is not open', (t) => {
+    const db = new DatabaseSync(nextDb(), { open: false });
+
+    t.assert.throws(() => {
+      db.location();
+    }, {
+      code: 'ERR_INVALID_STATE',
+      message: /database is not open/,
+    });
+  });
+
+  test('throws if provided dbName is not string', (t) => {
+    const db = new DatabaseSync(nextDb());
```

Comment:
I think this database needs to be closed. It will be really nice when we can use `using` for these.

Reply:
done